src/SmartComponents/AdvisorySystems/AdvisorySystem.test.js

```diff
@@ -8,8 +7,11 @@ import { initMocks } from '../../Utilities/unitTestingUtilities.js';
 import AdvisorySystems from './AdvisorySystems';
 import { BrowserRouter as Router } from 'react-router-dom';
 import NoRegisteredSystems from '../../PresentationalComponents/Snippets/NoRegisteredSystems';
+import { render, act as rtlAct } from '@testing-library/react';
```

Comment:
You can remove the other outdated not to be used anymore act, and not need to rename it.